packages/create-llama/helpers/python.ts

```diff
@@ -177,17 +179,22 @@ export const installPythonTemplate = async ({
       vectorDbDirName,
     );
     const enginePath = path.join(root, "app", "engine");
-
     await copy("**", path.join(root, "app", "engine"), {
       parents: true,
       cwd: VectorDBPath,
     });
-    let dataSourceDir = dataSource?.type ?? "file";
-    const loaderPath = path.join(compPath, "loaders", "python", dataSourceDir);
-    await copy("**", enginePath, {
-      parents: true,
-      cwd: loaderPath,
-    });
+
+    const dataSourceType = dataSource?.type;
+    if (dataSourceType !== undefined && dataSourceType !== "none") {
+      let loaderPath =
+        dataSourceType === "folder"
+          ? path.join(compPath, "loaders", "python", "file")
+          : path.join(compPath, "loaders", "python", dataSourceType);
```

Comment:
why we need a different loader for file or folder?

Reply:
this code already uses the same loader for both file and folder, it just maps `folder` to `file`